browser/ui/containers/containers_menu_model_unittest.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/containers/containers_menu_model.h"
+
+#include <vector>
+
+#include "base/strings/utf_string_conversions.h"
+#include "brave/browser/ui/containers/container_model.h"
+#include "brave/browser/ui/containers/mock_containers_menu_model_delegate.h"
+#include "testing/gmock/include/gmock/gmock.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+using containers::ContainerModel;
+using containers::ContainersMenuModel;
+
+class ContainersMenuModelUnitTest : public testing::Test {
```

Comment:
pls put the test into `containers` namespace and remove all mentions of `containers::`, including `using containers::` above.